capture_gui/lib.py

```diff
@@ -290,6 +289,17 @@ def default_output():
     return os.path.join(workspace, folder, filename)
 
 
+def get_project_rule(rule):
+    """Get the full path of the rule of the project"""
+    workspace = cmds.workspace(query=True, rootDirectory=True)
+    folder = cmds.workspace(fileRuleEntry=rule)
+    if not folder:
+        log.warning("File Rule Entry '{}' has no value, please check if the "
```

Comment:
Is this overly verbose? 

Technically a file rule entry could exist and be set to an empty value. I'd expect this kind of warning solely when the file rule entry didn't exist when being queried. Is there a way to identify whether the file rule exists?